.gitignore

```diff
@@ -1 +1,21 @@
-.*.sw?
+-.*.sw?
```

Comment:
Either the diff is odd, or the pattern has been changed to `-.*.sw?`; it should be `.*.sw?`.